options/posix/generic/posix_stdlib.cpp

```diff
@@ -469,3 +469,8 @@ int getloadavg(double *, int) {
 	__ensure(!"Not implemented");
 	__builtin_unreachable();
 }
+
+char *secure_getenv(const char *name) {
+	// We don't have a secure execution environment.
```

Comment:
Maybe at a log here that we don't implement it properly, and submit an issue when this is merged. It's not too difficult but not really needed for this PR.

Reply:
Yep, sounds good. I'll do that as soon as I have time.